systemd-mount: add page; systemd-umount: add alias page (#10509)
tldr-pages/tldr: pages/linux/systemd-mount.md

# systemd-mount

> Establish and destroy transient mount or auto-mount points.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-mount.html>.

> - Mount a file system (image or block device) and automatically determine the mount point:

Reviewer: ```suggestion
- Mount a file system (image or block device) at `/run/media/system/LABEL` where LABEL is the filesystem label or the device name if there is no label:
```

`systemd-mount {{path/to/file_or_device}}`

- Mount a file system (image or block device) at a specific location:

`systemd-mount {{path/to/file_or_device}} {{path/to/mount_point}}`

> - Unmount a specific mount point or device:
> 
> `systemd-mount --umount {{path/to/mount_point_or_device}}`
> 

Reviewer: ```suggestion
```

- Show a list of all local, known block devices with file systems that may be mounted:

`systemd-mount --list`

- Create an automount point that mounts the actual file system at the time of first access:

`systemd-mount --automount=yes {{path/to/file_or_device}}`

> - Unmount multiple mount points or devices:

Reviewer: ```suggestion
- Unmount one or more devices:
```

`systemd-mount --umount {{path/to/mount_point_or_device1}} {{path/to/mount_point_or_device2}}`

> - Mount a file system (image or block device) with specific file system type:

Reviewer: ```suggestion
- Mount a file system (image or block device) with a specific file system type:
```
